apps/passport-server/src/database/queries/telegram/fetchTelegramEvent.ts

```diff
@@ -37,19 +37,27 @@ export async function fetchTelegramEventsByChatId(
   return result.rows;
 }
 
-export async function fetchTelegramEvent(
-  client: Pool,
-  ticketEventId: string,
-  telegramChatId: number
-): Promise<TelegramEvent | null> {
+export interface LinkedPretixTelegramEvent {
+  telegramChatID: string | null;
+  eventName: string;
+  configEventID: string;
+}
+
+export async function fetchLinkedPretixAndTelegramEvents(
+  client: Pool
+): Promise<LinkedPretixTelegramEvent[]> {
   const result = await sqlQuery(
     client,
     `\
-    select * from telegram_bot_events
-    where ticket_event_id = $1 and telegram_chat_id = $2
-    `,
-    [ticketEventId, telegramChatId]
+    SELECT
+      tbe.telegram_chat_id AS "telegramChatID",
+      dpe.event_name AS "eventName",
+      dpe.pretix_events_config_id AS "configEventID" 
+    FROM pretix_events_config pec
```

Comment:
I think we can remove the pretix_events_config from this query and just join directly on the IDs

Reply:
Done